### PR TITLE
Project 3: Contact form (pair programming)

### PR DESCRIPTION
**Pull request by Paul-js-hub**
### Portfolio Project 3
In this project we:

- Added markup and styling to index and style files
- Added margin, active, and hover effects to the button
- Added footer to the project
- Modified index and style files so that the mobile should be responsive

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -204,11 +204,60 @@ <h1>Multi-Post Stories</h1>
...
           <button class="seeproject-btn" type="submit">See Project</button>
         </div>
       </section>
+      <form action="https://formspree.io/f/xqkngpry" method="post">
+        <h1>CONTACT US</h1>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
```
Comment (williamrolando88) [inline index.html]: - [ ] These elements should be outside the `form` tag. Is recommendable to only locate form elements inside a form tag

File: index.html [hunk 2]
```diff
@@ -204,11 +204,60 @@ <h1>Multi-Post Stories</h1>
...
               <span> <img src="images/image3.png" alt="bootstrap" /></span>
             </li>
-            <li>Ruby</li>
+            <li class="list-sumbua">Ruby</li>
           </ul>
           <button class="seeproject-btn" type="submit">See Project</button>
         </div>
       </section>
+      <form action="https://formspree.io/f/xqkngpry" method="post">
+        <h1>CONTACT US</h1>
```
Comment (williamrolando88) [inline index.html]: - [ ] You can only use ONE `h1` tag per HTML document, the next header always must be `h2` - `h6`. Please change the tag according to your hierarchy.

File: index.html [hunk 3]
```diff
@@ -204,11 +204,60 @@ <h1>Multi-Post Stories</h1>
...
+        />
+        <textarea
+          name="textarea"
+          class="text-area"
+          maxlength="500"
+          placeholder="Enter text here"
+          required
+        ></textarea>
+        <button class="seeproject-btn" type="button">Submit</button>
+      </form>
```
Comment (williamrolando88) [inline index.html]: - [ ] This block of code must be wrapped in a `section` tag. The form is only one of the elements within that section. Please fix this issue

File: index.html [hunk 4]
```diff
@@ -204,11 +204,60 @@ <h1>Multi-Post Stories</h1>
...
+          required
+        />
+        <textarea
+          name="textarea"
+          class="text-area"
+          maxlength="500"
+          placeholder="Enter text here"
+          required
+        ></textarea>
+        <button class="seeproject-btn" type="button">Submit</button>
```
Comment (williamrolando88) [inline index.html]: - [ ] Your form currently doesn't work, this is due to this button, for the form to work type must be `type="submit"`. Please fix this issue

File: index.html [hunk 5]
```diff
@@ -204,11 +204,60 @@ <h1>Multi-Post Stories</h1>
...
+          name="email"
+          required
+        />
+        <textarea
+          name="textarea"
+          class="text-area"
+          maxlength="500"
+          placeholder="Enter text here"
+          required
+        ></textarea>
```
Comment (williamrolando88) [inline index.html]: - [ ] According to your Figma template, you need to add padding in your `input` fields and `textarea`. 

Figma template:
image

Your project looks like this:
image

**Review by williamrolando88 — CHANGES_REQUESTED** [review]
Hi @Paul-js-hub, @BwalyaNonde 

### Good job so far!  👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Nice README file
- Nice PR title and description
- Nice commit history

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] [Optional] I strongly suggest you avoid using width and height fixed values for your styling. For the purpose of this exercise, you must create a responsive design, and using fixed values for those properties is counterproductive.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification, also you can send me a Slack message.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @Paul-js-hub  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**